spaghetto/bin/src/Main.purs

```diff
@@ -68,6 +72,20 @@ type InstallArgs =
   , output :: Maybe String
   , pedanticPackages :: Boolean
   , ensureRanges :: Boolean
+  , jsonErrors :: Boolean
+  , psaArgs :: PsaArgs
+  }
+
+type PsaArgs =
```

Comment:
There already is a `PsaOptions` type in the Psa source so there's no need to bundle these together here, and I think it'd make more sense to group by functionality, e.g. by calling this something like `BuildFlags`, and adding here all the other build-related things (as grouped in the configuration file) like `output`, `pedanticPackages`, `jsonErrors`, etc